api/envoy/data/core/v2alpha/local_reply_body.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package envoy.data.core.v2alpha;
+
+option java_outer_classname = "LocalReplyBodyProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.data.core.v2alpha";
+
+// [#protodoc-title:  local reply body]
+// Used to wrap a local response, and then converted to json format to return.
+message LocalReplyBody {
```

Comment:
I think maybe call this `LocalReply` since eventually we might want to include other things in here?

Reply:
Agree